lib/ctbl.gi

```diff
@@ -1943,12 +1952,10 @@ InstallMethod( ClassPositionsOfMinimalNormalSubgroups,
           minimal,   # list of minimal kernels
           k;         # one kernel
 
-    # Every normal subgroup is an intersection of kernels of characters,
-    # so maximal normal subgroups are kernels of irreducible characters.
-    normal:= Set( ClassPositionsOfNormalSubgroups( tbl ) );
+    normal:= Set( List( [ 2 .. NrConjugacyClasses( tbl ) ],
```

Comment:
Minor remark: `Set( List( l, f ) )` is equivalent to (but slower than) `Set( l, f )`

Reply:
@fingolfin Interesting, the two argument version of `Set` (resp. `SSortedList`) seems to be undocumented. Should we document it?

Reply:
Ah, I was not aware that it is undocumented. Yes, I think it should be documented!